components/style/properties/properties.mako.rs

```diff
@@ -868,8 +916,10 @@ impl PropertyId {
 /// Includes shorthands before expansion
 pub enum ParsedDeclaration {
     % for shorthand in data.shorthands:
+        % if shorthand.name != "all":
```

Comment:
This and the other one can't use `shorthands_except_all` since it only surrounds part of the for loop body.